deps/jerry/jerry-core/CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ set(FEATURE_MEM_STATS          OFF      CACHE BOOL   "Enable memory statistics?"
 set(FEATURE_MEM_STRESS_TEST    OFF      CACHE BOOL   "Enable mem-stress test?")
 set(FEATURE_CPU_PROFILER       OFF      CACHE BOOL   "Enable Cpu profiler?")
 set(FEATURE_HEAP_PROFILER      OFF      CACHE BOOL   "Enable Heap profiler?")
-set(FEATURE_FUNCTION_NAME      OFF      CACHE BOOL   "Enable saving Javascript function name?")
+set(FEATURE_DEBUG_INFO         ON       CACHE BOOL   "Enable saving Javascript debug info?")
```

Comment:
Suggestion name: FEATURE_EXTRA_INFO, the _DEBUG_ might be confusion with the jerry-debugger and the define `JERRY_NDEBUG`.

Reply:
EXTRA_INFO is too generic. DEBUG_SYMBOL is ok?

Reply:
Good to me :)

Reply:
If DEBUG_SYMBOL is still confusing with NDEBUG, is SOURCE_SYMBOL ok?